### PR TITLE
Completed css-pseudo-classes exercise.- Devan Rivera 

### PR DESCRIPTION
**Pull request by devanrivera98**
Screen Shot 2023-01-25 at 10 46 27 AM

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+button {
+  width: 15%;
+  padding: 5px 10px;
+}
+
+button:hover {
+  cursor: pointer;
+}
```
Comment (shawnkost) [inline css-pseudo-classes/styles.css:8]: The `background-color` should change to black once the button has been hovered

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Devan! One small change requested below. Make sure you're comparing your output to the GIF included in the instructions

**Comment by devanrivera98** [comment]
Screen Shot 2023-01-25 at 11 22 21 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**